src/Theme.tsx

```diff
@@ -0,0 +1,21 @@
+const colors = {
+  white: '#fff',
+  cream: '#f7f9fc',
+  dark: '#25262b',
+  gray: '#ccc',
+  orange: '#ff7b3a',
+  lightOrange: '#e59560',
+  blue: '#3498db',
+};
```

Comment:
변하지 않는 상수와 같은 경우에는 끝에 as const를 붙여주시면 타입을 더 strict하게 할 수 있답니다
```suggestion
} as const;
```

Reply:
Theme.tsx, Mbti.tsx 파일에 as const 타입을 추가하였습니다!